cmd/users.go

```diff
@@ -128,13 +132,28 @@ var cmdDeploymentLogin = &cobra.Command{
 			log.Fatal(err)
 		}
 
-		resp, err := deployment.LogIn(username, string(pwBytes))
+		resp, err := deployment.LogIn(username, string(pwBytes), "")
```

Comment:
TOTP is one-time use right? Is it safe to allow users to provide it via command line flag if they want to?

Reply:
Yea I think that's fine.